src/mono/wasm/runtime-test.js

```diff
@@ -115,7 +115,7 @@ setenv = {};
 runtime_args = [];
 enable_gc = true;
 enable_zoneinfo = false;
-while (true) {
+while (args.length > 0) {
```

Comment:
if args is actually undefined this will be an error won't it?

Reply:
I have pushed a commit to ensure that it will never be undefined, but also added a check for `while`, just in case :D